cloudbuild.yaml

```diff
@@ -31,86 +31,28 @@ steps:
     timeout: 5m
     args: ['./scripts/git_init']
 
-  # Make sure that the Bazel server is unpacked and initialized, by calling `bazel info` (any other
-  # Bazel command would probably do).
   - name: 'gcr.io/oak-ci/oak:latest'
-    id: bazel_init
-    entrypoint: 'bazel'
+    id: runner_ci
     waitFor: ['git_init']
-    timeout: 5m
-    args: ['info']
-
-  - name: 'gcr.io/oak-ci/oak:latest'
-    id: cargo_crev
-    entrypoint: 'bash'
-    waitFor: ['git_init']
-    timeout: 5m
-    args: ['./scripts/run_cargo_crev']
-
-  - name: 'gcr.io/oak-ci/oak:latest'
-    id: build_server_no_debug
-    waitFor: ['bazel_init']
-    timeout: 60m
-    entrypoint: 'bash'
-    args: ['./scripts/build_server', '-s', 'logless']
-
-  - name: 'gcr.io/oak-ci/oak:latest'
-    id: build_server
-    waitFor: ['build_server_no_debug']
     timeout: 60m
     entrypoint: 'bash'
-    args: ['./scripts/build_server', '-s', 'base']
+    args: ['./scripts/runner', '--commands', 'run-ci']
 
   # Package the Hello World application in a Docker image.
   - name: 'gcr.io/oak-ci/oak:latest'
     id: build_server_docker
-    waitFor: ['build_server']
-    timeout: 60m
-    entrypoint: 'bash'
-    args: ['./scripts/build_example', '-e', 'hello_world', '-i', 'base']
-
-  - name: 'gcr.io/oak-ci/oak:latest'
-    id: run_tests
-    waitFor: ['build_server_docker']
-    timeout: 60m
-    entrypoint: 'bash'
-    args: ['./scripts/run_tests']
```

Comment:
`run_tests` includes a `cargo test --doc` step that doesn't seem to be in `runner` &ndash; I guess that's already covered by `cargo test` ?